uiautomationutilities/.gitignore

```
@@ -1 +1,3 @@
 /build
+/perf_tool/.vs/
+/perf_tool/obj/
```

```suggestion
/perf_tool/obj/

```

(adding newline for EOF/posix compat)